raiden/app.py

```diff
@@ -95,11 +95,12 @@ def __init__(
         )
 
         # check that the settlement timeout fits the limits of the contract
-        invalid_timeout = (
+        invalid_settle_timeout = (
             config['settle_timeout'] < default_registry.settlement_timeout_min() or
-            config['settle_timeout'] > default_registry.settlement_timeout_max()
+            config['settle_timeout'] > default_registry.settlement_timeout_max() or
+            config['settle_timeout'] < config['reveal_timeout'] // 2
```

Comment:
Why `config['reveal_timeout'] // 2`? Should'n it be at least a whole reveal timeout?

Reply:
AHHHH, this is wrong, nice catch, it should be at least twice the reveal timeout.

We need at least one reveal timeout otherwise a transfer will never be accepted, because it's always inside the danger zone, additionally we need enough time for the protocol to complete